### PR TITLE
[docs] Update README with BYOH removal info

### PR DESCRIPTION
**Pull request by sebsoto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +56,11 @@ data:
     username=core
 ```
 
+#### Removing BYOH Windows instances
+BYOH instances that have been joined to the cluster as a node can be removed by deleting the instance's entry in the
+ConfigMap. In order for the instance to be cleanly removed, it must be accessible with the current private key provided
+to WMCO.
```
Comment (aravindhp) [inline README.md:62]: Please provide an example by removing `10.1.42.1`

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for doing this, @sebsoto

File: README.md [hunk 2]
```diff
@@ -57,6 +56,11 @@ data:
     username=core
 ```
 
+#### Removing BYOH Windows instances
+BYOH instances that have been joined to the cluster as a node can be removed by deleting the instance's entry in the
```
Comment (EricPonvelle) [inline README.md:60]: I'm unsure if "joined" is a term, but if it's not, maybe "BYOH instances that are attached to the cluster..."

**Review by EricPonvelle — CHANGES_REQUESTED** [review]
I have one small correction.

**Review by EricPonvelle — APPROVED** [review]
/lgtm

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

**Review by aravindhp — APPROVED** [review]
/approve

**Comment by mansikulkarni96** [comment]
/ cherry-pick release-4.8

**Pull request merged**